pages/pipelines/links_and_images_in_log_output.md.erb

```diff
@@ -1,12 +1,40 @@
-# Images in Log Output
+# Links & Images in Log Output
 
-You can embed external images or images generated by your builds, directly into your logs using a special ANSI escape code.
+You can embed links, external images or images generated by your builds, directly into your logs using special ANSI escape codes.
+
+ANSI escape codes are used for many tasks in UNIX terminals, for example, setting text color uses an ANSI escape code. Buildkite supports most standard, and a few special ones, thanks to our terminal output processor, [`terminal-to-html`](https://github.com/buildkite/terminal-to-html).
 
 <%= toc %>
 
-## Syntax for Inlining Images
+## Links
+
+You can embed clickable links, to Buildkite or other web pages, via ANSI escape code `1339`.
+
+The following Bash function takes a URL and optional link text and will output the correct escape code sequence:
+
+```bash
+function inline_link {
+  LINK="url='"$1"'"
+
+  if [ $# -gt 1 ]; then
+    LINK="$LINK;content='"$2"'"
+  fi
+
+  printf '\033]1339;'$LINK'\a\n'
+}
+```
+
+You can use it like so:
+
+```bash
+inline_link "https://buildkite.com/"
+```
```

Comment:
Maybe also worth mentioning that you can link artifacts using "artifact://..." urls?

Reply:
Done!

Reply:
Nice :+1: